app/src/main/res/values/strings.xml

```diff
@@ -87,185 +87,186 @@
     <string name="action_show_solution">Show solution</string>
     <string name="action_show_mine_sudoku">Show mine</string>
     <string name="onboard_title">Welcome</string>
-    <string name="intro_what_is_sudoku">A classic sudoku looks as follows</string>
-    <string name="intro_rules">The goal of the game is to fill all the cells with numbers from 1 to 9. So that the values do not repeat in any column, row or smaller box (3x3)</string>
-    <string name="onboard_recommendation_prefs">We recommend that you have a look at the settings tab so that you can customize the application for your needs</string>
+    <string name="intro_what_is_sudoku">A classic sudoku</string>
+    <string name="intro_rules">Fill cells using 1–9 without repetition in any column, row, or smaller (3x3) box.</string>
+    <string name="onboard_recommendation_prefs">Adjust the settings to your liking.</string>
     <string name="action_start">Start</string>
-    <string name="toolbar_undo_description">Undo. Just undo</string>
-    <string name="toolbar_notes_description">Activates the note mode (pencil marks)</string>
-    <string name="toolbar_hint_description">Inserts the correct value in the selected cell. Can be disabled in settings</string>
-    <string name="toolbar_erase_description">Removes a value or all notes from the selected cell</string>
+    <string name="toolbar_undo_description">Undo. Just undo.</string>
+    <string name="toolbar_notes_description">Note mode (pencil marks)</string>
```

Comment:
https://m1.material.io/style/writing.html#writing-language
>Pick common words that are clearly and easily understandable to both beginning and advanced English readers. 

It suggests "turn on" over "enable". Activate is similarly less common.
If "Crash reporting" "mistake limit" and "Mistake checking" can be understood to be turned on without it saying so/without associated verb, then this is above board (?).

Reply:
I think it is useful, as it gives an explanation of the icons.
This helps tie something to them, or explain them if the menu is sought out because it is unclear.

What note mode actually does remains unexplained. Whether it gets turned on by clicking the button is not bringing any clarity that I can see a scenario for.